Fix Issue #13671: Make sections consistent across modules (#15054)
mdn/content: files/en-us/learn/html/tables/advanced/index.md

## Summary

> There are a few other things you could learn about tables in HTML, but we have really given all you need to know at this moment in time. At this point, you can go on to testing yourself with our [table assessment](/en-US/docs/Learn/HTML/Tables/Structuring_planet_data). Have fun!

Reviewer: ```suggestion
There are a few other things you could learn about tables in HTML, but this is all you need to know for now. Next, you can test yourself with our [table assessment](/en-US/docs/Learn/HTML/Tables/Structuring_planet_data). Have fun!
```

If you are already learning CSS and have done well on the assessment, you can move on and learn about styling HTML tables — see [Styling tables](/en-US/docs/Learn/CSS/Building_blocks/Styling_tables).

> You want to get started with learning CSS? We recommend you to check out the [CSS Learning Area](/en-US/docs/Learn/CSS)!

Reviewer: ```suggestion
If want to get started with learning CSS, check out the [CSS Learning Area](/en-US/docs/Learn/CSS)!
```

Author: better, thanks 👍
